pyptex/__init__.py

```diff
@@ -565,35 +585,28 @@ def pyptexmain(argv=None):
     """
     argv = argv or sys.argv
     dopdb = True
-    try:
-        k = argv.index('--pdb=no')
-        argv.pop(k)
+    with suppress(ValueError):
```

Comment:
I worry ValueError is not the only Exception that can arise here and need to be suppressed? The idea is that argv should pass through as unmolested as possible if anything goes wrong. I dunno, an exception is generated because something isn't hashable or comparable or can't be popped for whatever reason?
```suggestion
    with suppress(Exception):
```